src/Activity/TrainWithExercises/ShowExercise.js

```diff
@@ -76,8 +79,16 @@ class ShowExercise extends React.Component {
       }
     } else if (this.state.studentAnswer) {
       bubbleText = this.props.student.givePositiveAnswer;
+      if (this.props.activityChosen === "mammals"){
+        bubbleText = this.props.student.givePositiveAnswerMammal;
+        console.log(bubbleText)
```

Comment:
Do not leave `console.logs` in your code. It is useful for debugging, but it will become very messy if when we have console.logs everywhere